### PR TITLE
Code Refactoring

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
[Badge Suggestions]: https://img.shields.io/badge/Ｒｅｐｏｓｉｔｏｒｙ_Ｓｕｇｇｅｓｔｉｏｎｓ-4b781e?style=for-the-badge&logoColor=white&logo=GitHub&labelColor=609926
[Badge Changes]: https://img.shields.io/badge/Ｃｈａｎｇｅｓ-9e122a?style=for-the-badge&logoColor=white&logo=GitLFS&labelColor=D81939
[Badge GitHub]: https://img.shields.io/badge/ＧｉｔＨｕｂ_Ｎｅｗｃｏｍｅｒｓ-be2e47?style=for-the-badge&logoColor=white&logo=BitWarden&labelColor=FD3A5C
[Badge Robot]: https://img.shields.io/badge/Ｉ＇ｍ_Ｈｕｍａｎ-8c3b63?style=for-the-badge&logoColor=white&logo=RuboCop&labelColor=c84f8c
[Badge Ask]: https://img.shields.io/badge/Ｐｌｅａｓｅ_Ａｓｋ_Ｍｅ-4e2760?style=for-the-badge&logoColor=white&logo=AskUbuntu&labelColor=73398D
[Badge Info]: https://img.shields.io/badge/Ｉｎｆｏｒｍａｔｉｏｎ-01434e?style=for-the-badge&logoColor=white&logo=InternetArchive&labelColor=006272

[Button MarkedDown]: https://img.shields.io/badge/MarkedDown-00b2ff?style=for-the-badge&logoColor=white&logo=Markdown

[License Badges]: https://github.com/MarkedDown/Licenses
[Widgets]: https://github.com/MarkedDown/Disable/blob/main/Features/Widgets.md
[Tabs]: https://github.com/MarkedDown/Disable/blob/main/Features/Tabs.md
[Wiki]: https://github.com/MarkedDown/Wiki
[MarkedDown]: https://github.com/MarkedDown 'How to better use Markdown'

<br>
<br>

<div align = center>

[<kbd> <br> Please Check The **Preview** <br> </kbd>][Preview]

</div>

<br>
<br>

## ![Badge Changes]

- General formatting / refactoring

- Separated commands

- Replaced dynamic os dependent commands
  with static predetermined ones.

<br>
<br>

## ![Badge Ask]

- ***If you have any questions.***

- ***In case you do not agree with some of the changes made in this PR***,
  *please first let me know about them, before disregarding it entirely so*
  *we can possibly find a middle ground and have it not go to waste.*

<br>
<br>

## ![Badge Robot]

<kbd> <br>            No, I'm not a robot.            <br> </kbd>

<kbd> <br>    Yes, you are not the first.    <br> </kbd>

<kbd> <br>     I just made a PR template.     <br> </kbd>

<br>
<br>

[Preview]: https://github.com/ElectronicsArchiver/deno_process/tree/Code-Refactoring

### CODE REVIEW

**Comment by ElectronicsArchiver** [comment]
@axetroy Deno formatting bs again~

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
+                deno : ['v1.x']
+                os : [ ubuntu-latest , macOS-latest , windows-latest ]
+        
+        steps:
+            
+            -   uses : actions/checkout@v2
+            
+            -
+                name : Setup deno
+                uses : denolib/setup-deno@v2
```
Comment (axetroy) [inline .github/workflows/test.yml]: Could you replace it with Deno official action?

**Comment by axetroy** [comment]
There is a problem, the directory starts with an uppercase.

I don't think it's recommended

File: .github/workflows/test.yml (hunk 1 above)
Comment (ElectronicsArchiver) [reply .github/workflows/test.yml]: Didn't change it, but sure~

**Comment by ElectronicsArchiver** [comment]
> There is a problem, the directory starts with an uppercase.
> 
> I don't think it's recommended

:laughing:

Only badly designed environments force you to use one over the other.

For this, there is absolutely no problem~

**Comment by ElectronicsArchiver** [comment]
Again, I would recommend not using the deno formatting considering that little is covered by the editorconfig..

**Comment by axetroy** [comment]
Thank you very much for your PR, And I have a rough preview, there are a few points that I may not want to merge:

1. Directory and file naming is not quite what I expected
2. Typescript has been rewritten with javascript
3. Not use Deno formatter. It's not that using editorconfig doesn't work, it's just that there is no CI to go with it

I believe that the constraints of tools are far greater than the review of people.

So I stuck with Typescript/Deno formatter/Deno lint .etc

**Pull request closed without merging**